### DISCUSSION TITLE
What's the syntax for CONTAINER_EXPIRATION_POLICY_ATTRIBUTES ?

### DISCUSSION THREAD

**Discussion by KarHo17** [python-gitlab/python-gitlab#2284] [General]
I'm trying to figure out the syntax for the gitlab cli to set the container expiration policy attributes for my projects.

gitlab project update --id \<ID> --container-expiration-policy-attributes CONTAINER_EXPIRATION_POLICY_ATTRIBUTES.

I've tried yaml syntax, python syntax, even writing the data in a file and reading it back with @/tmp/filename.
Each of my attempts resulted in 'Impossible to update object (400: container_expiration_policy_attributes is invalid)'

Does anyone know the correct syntax to set the policy attributes per CLI ?
It was easy to do with the gitlab python module, but I'm sure that I could accomplish the same with the CLI.

**Comment by nejch** [comment]
@KarHo17 I think that's a hash type and unfortunately we don't formally support `hash` types in the CLI yet, we need to add some parsing there. Could you please subscribe to this issue for progress on that? https://github.com/python-gitlab/python-gitlab/issues/1735
